web-app/src/components/Footer.js

```diff
@@ -17,6 +17,11 @@ const Footer = () => {
           links={[
             { title: "Terms of Use", href: "/terms" },
             { title: "Privacy policy", href: "/privacy" },
+            {
+              title: "Code Contribution",
+              href:
+                "https://github.com/lebanon-relief-project/LRP-web-app/blob/master/CONTRIBUTING.md",
```

Comment:
I am not sure whether I like the url being stored inside the component itself, maybe we should have a statics file or env variable instead. This would also allow us to point to the dev version of the document when we are viewing the dev app.

Reply:
I don't think this will be a problem for the foreseeable future - we should merge this and worry about this if we need that functionality